x-pack/plugins/reporting/public/management/report_info_button.tsx

```diff
@@ -287,3 +298,5 @@ export class ReportInfoButton extends Component<Props, State> {
     }
   };
 }
+
+export const ReportInfoButton = injectI18n(ReportInfoButtonUi);
```

Comment:
This component has been changed to show a tooltip when the cursor hovers over the `info` icon. This makes it more consistent with the `download` button.